skimage/filters/tests/test_gabor.py

```diff
@@ -77,6 +77,18 @@ def match_score(image, frequency):
     assert responses[1, 1] > responses[1, 0]
 
 
+def test_morlet_zero_sum():
```

Comment:
that should be fine. I guess to speed it up a little, we could reduce the max range here from 10 to 6?